crates/next-core/src/next_font/google/font_fallback.rs

```diff
@@ -22,43 +21,51 @@ use crate::{
 };
 
 /// An entry in the Google fonts metrics map
-#[derive(Deserialize, Debug)]
+#[derive(Deserialize, Serialize, Debug, PartialEq, Eq, TraceRawVcs, NonLocalValue)]
 #[serde(rename_all = "camelCase")]
 pub(super) struct FontMetricsMapEntry {
     category: RcStr,
     ascent: i32,
     descent: i32,
     line_gap: u32,
     units_per_em: u32,
-    x_width_avg: f64,
+    x_width_avg: u64,
 }
 
-#[derive(Deserialize, Debug)]
-pub(super) struct FontMetricsMap(pub FxHashMap<RcStr, FontMetricsMapEntry>);
+#[derive(Debug)]
+#[turbo_tasks::value]
+pub(super) struct FontMetricsMap(pub FxIndexMap<RcStr, FontMetricsMapEntry>);
 
 #[derive(Debug, PartialEq, Serialize, Deserialize, TraceRawVcs, NonLocalValue)]
 struct Fallback {
     pub font_family: RcStr,
     pub adjustment: Option<FontAdjustment>,
 }
 
+#[turbo_tasks::function]
+async fn load_font_metrics(project_root: FileSystemPath) -> Result<Vc<FontMetricsMap>> {
+    let data: FontMetricsMap = load_next_js_templateon(
```

Comment:
afaict the file read is cached, but not the json parse.

so is it possible they are just CPU constrained rather than filesystem constrained?  

a good fix either way!